libraries/tokens/src/types/index.ts

```diff
@@ -28,10 +28,12 @@ export type TypographyTokens = {
 }
 
 export type Spacing = {
-  left: string
-  right: string
+  left?: string
+  right?: string
   top?: string
   bottom?: string
+  x?: string
+  y?: string
```

Comment:
If we are adding new properties to `Spacing` type you need to also update the [spacingsTemplate](https://github.com/equinor/design-system/blob/c6433df9f32fbb2c88bca0e768eda58940551baf/libraries/core-react/src/utils/templates/index.ts#L51) to support  these properties to avoid bugs in the future where people think they are supported in the template and aren't.  

I would also suggest maybe adding a secondary parameter to the template which specifies spacings `type?: "padding" | "margin"`. It be set to `"padding"`  as default value.

Reply:
> I would also suggest maybe adding a secondary parameter to the template which specifies spacings `type?: "padding" | "margin"`. It be set to `"padding"` as default value.

Let's see if we need it. This is probably a bigger change in a lot of components, might be a separate issue. 🤔

Reply:
I removed the x and y options as it probably complicate things. Changed the `margin` to `padding` in the Breadcrumb, it's not exactly wrong IMHO. But we might consider your suggestion of a better separation between margin and padding, but let's do that later.